src/main/scala/it/pagopa/interop/catalogmanagement/model/persistence/serializer/v1/utils.scala

```diff
@@ -145,8 +151,11 @@ object utils {
       dailyCallsTotal = ver1.dailyCallsTotal,
       agreementApprovalPolicy = agreementApprovalPolicy,
       createdAt = createdAt.getOrElse(defaultCreatedAt),
-      activatedAt = if (state == Draft) None else activatedAt orElse defaultActivatedAt.some,
-      serverUrls = ver1.serverUrls.toList
+      serverUrls = ver1.serverUrls.toList,
+      publishedAt = if (state == Draft) None else publishedAt orElse defaultPublishedAt.some,
+      suspendedAt = if (state == Draft) None else suspendedAt,
+      deprecatedAt = if (state == Draft) None else deprecatedAt,
+      archivedAt = if (state == Draft) None else archivedAt
```

Comment:
I think that this could just be
```suggestion
      suspendedAt = suspendedAt,
      deprecatedAt = deprecatedAt,
      archivedAt = archivedAt
```